src/routes/v1/articles.routes.js

```diff
@@ -0,0 +1,21 @@
+import dotenv from 'dotenv';
+import { Router } from 'express';
+import RatingsController from '../../controllers/ratings.controller';
+import authMiddleware from '../../middleware/auth.middleware';
+import validateRatings from '../../middleware/validation.middleware';
+
+// middleware
+const validateRequest = validateRatings();
+
+dotenv.config();
+
+const route = Router();
+
+route.patch(
+  '/:id/ratings',
+  authMiddleware,
+  validateRequest,
+  RatingsController.rateArticles
+);
```

Comment:
Firstly, I think `validateRequest` should come before `authMiddleware`. 

Secondly, To rate an article, do I actually need to login? I'm wondering if this route actually need to be protected.

Reply:
I believe we don't just anybody that we can't trace their identity to just rate an article, there should be a way to detect those who rate an article and that should be through the decoding their user information through the token provided that grant the user access to that route.